packages/@react-aria/dnd/src/useDropIndicator.ts

```diff
@@ -90,6 +91,7 @@ export function useDropIndicator(props: DropIndicatorProps, state: DroppableColl
     // If aria-hidden, we are either not in a drag session or the drop target is invalid.
     // In that case, there's no need to render anything at all unless we need to show the indicator visually.
     // This can happen when dragging using the native DnD API as opposed to keyboard dragging.
-    isHidden: !isDropTarget && !!ariaHidden
+    isHidden: !isDropTarget && !!ariaHidden,
+    isVirtualDrag: !!dragSession
```

Comment:
Needed some way to tell the ListViewItem to override the `tabIndex` provided by useGridListItem. Couldn't really put it into useGridListItem itself since DnD behavior is handled separately. Didn't want to set `tabIndex=null` in useDropIndicator/useDroppableItem since the `tabIndex=null` behavior is specific to the grid setup of ListView and not so much to a general drop indicator/droppable item, so opted for having useDropIndicator inform the user whether or not a keyboard/virtual drag session is happening. Open to opinions here, it might be an idea to have the other DnD hooks return `isVirtualDrag` as well? Or some other way to know if a drag session is ongoing?